README.md

```diff
@@ -1 +1,2 @@
-# django-girls-blog-DrEdi
\ No newline at end of file
+# django-girls-blog-DrEdi
+https://travis-ci.org/kpi-web-guild/django-girls-blog-DrEdi.svg?branch=master
```

Comment:
There's a markdown version of the button showing image and making it a link. Use it